subprojects/test-runner/client/src/main/kotlin/com/avito/runner/scheduler/listener/ArtifactsTestListener.kt

```diff
@@ -63,12 +63,20 @@ internal class ArtifactsTestListener(
                 }
                 TestResult.Complete(artifacts)
             }
+
             is InfrastructureError ->
-                TestResult.Incomplete(result)
+                TestResult.Incomplete(
+                    infraError = result,
+                    logcat = device.logcat(LOGCAT_LINES_TO_FETCH)
```

Comment:
```suggestion
                    logcat = device.logcat(LOGCAT_TOP_LINES)
```
The current behavior is obvious. Let's explain also the motivation.